src/main/kotlin/nl/info/zac/identity/model/ZACRole.kt

```diff
@@ -5,5 +5,13 @@
 package nl.info.zac.identity.model
 
 enum class ZACRole(val value: String) {
```

Comment:
```suggestion
enum class ZacApplicationRole(val value: String) {
```

Because application role is the term we use in the new IAM architecture, so this makes it a bit more clear what this is.